src/MSBuild/CommandLineSwitches.cs

```diff
@@ -119,6 +119,7 @@ internal enum ParameterizedSwitch
             GetTargetResult,
             GetResultOutputFile,
             FeatureAvailability,
+            MultiThreaded,
```

Comment:
For discussion: do we want to make it a cli parameter right now? At this point of development, a trait for the feature would be probably enough.